cli/client/response.go

```diff
@@ -98,16 +99,19 @@ func JSONBytesToArray(bytes []byte) ([]string, error) {
 	return array, err
 }
 
-// SortAndPrettyPrintJSONArray unmarshals an array of JSON strings, sorts and counts it and
-// returns a JSON marshaled version of it for pretty printing.
-func SortAndPrettyPrintJSONArray(bytes []byte) (bool, []byte, error) {
-	// Unmarshal into a []string so that we can count and sort the array
-	convertedArray, err := JSONBytesToArray(bytes)
-	if err != nil {
-		return false, nil, err
+// PrettyPrintSlice takes a slice (e.g. [0 1 2]), sorts it and returns a pretty printed string
+// in the same style as a JSON string array (e.g. ["0", "1", "2"]).
+func PrettyPrintSlice(slice []string) string {
+	sort.Strings(slice)
+	var buf bytes.Buffer
+	buf.WriteString("[")
+	for index, element := range slice {
```

Comment:
This would be cleaner:
```
buf.WriteString("[")
var bits []string
for _, element := range slice {
  bits = append(bits, fmt.Sprintf(...)
}
buf.WriteString(strings.Join(bits, ", ")
buf.WriteString("]")
return buf.String()
```